py_zipkin/logging_helper.py

```diff
@@ -159,17 +167,33 @@ def log_spans(self):
                 self.thrift_endpoint,
             )
 
+            if self.report_root_timestamp:
+                timestamp = self.start_timestamp
+                duration = server_end_time - self.start_timestamp
+            else:
```

Comment:
Cool. When I was writing this I realized that our current implementation is a bit limited - it doesn't allow generating and logging _only_ a client span. I.e. the outermost `zipkin_span` contextmanager is always going to report ('sr', 'ss').

This is _probably_ okay, since you'll only really ever log client spans in the context of another span. But in this context it means that we don't need to check for 'sr', because the outermost `zipkin_span` (i.e. the one with the logging_context) is always going to log it.

Writing this patch made me want to redesign this library a bit :-/